src/EnergyPlus/WaterThermalTanks.hh

```diff
@@ -663,7 +664,7 @@ namespace WaterThermalTanks {
               OffCycParaFuelRate(0.0), Mode(0), SaveMode(0), SaveWHMode(0), BackupElementCapacity(0.0), DXSysPLR(0.0),
               ReclaimHeatingSourceIndexNum(0), ReclaimHeatingSource(0), SetPointError(0), SetPointErrIndex1(0), IterLimitErrIndex1(0),
               IterLimitExceededNum1(0), RegulaFalsiFailedIndex1(0), RegulaFalsiFailedNum1(0), IterLimitErrIndex2(0), IterLimitExceededNum2(0),
-              RegulaFalsiFailedIndex2(0), RegulaFalsiFailedNum2(0)
+              RegulaFalsiFailedIndex2(0), RegulaFalsiFailedNum2(0), FirstTimeThroughFlag(false)
```

Comment:
This should be set to true? And added to clear_state.

Reply:
Ok I set it to be true. And for the ``clear_state``, would the "WaterHeaterDesuperheater.deallocate();" take care of that?

Reply:
Duh. Yes deallocating the desuperheater struct will take care of that.